### PR TITLE
Change current user

### PR DESCRIPTION
**Pull request by pavlokolodka**
https://trello.com/c/5jLo2fPq

### CODE REVIEW

File: src/components/profile/Profile.tsx [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
+import ProfileName from './profile-name';
 
 export const Profile: React.FC = () => {
-  const { user } = useContext(CurrentUserContext)
+  const { user, setUser } = useContext(CurrentUserContext)
   const navigate = useNavigate();
 
+  useEffect(() => {
+    const getUserWithVolunteer = async () => {
+      const userWithVolunteerandOrders = await userService.getUserWithVolunteerAndOrders();
```
Comment (RamboUA) [inline src/components/profile/Profile.tsx:19]:
```suggestion
      const userWithVolunteerAndOrders = await userService.getUserWithVolunteerAndOrders();
```

**Review by RamboUA — APPROVED** [review]
Thanks just a small fix, before you merge:

**Review by lerik1408 — COMMENTED** [review]
GJ!

File: src/components/profile/Profile.tsx [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
 
 export const Profile: React.FC = () => {
-  const { user } = useContext(CurrentUserContext)
+  const { user, setUser } = useContext(CurrentUserContext)
   const navigate = useNavigate();
 
+  useEffect(() => {
+    const getUserWithVolunteer = async () => {
+      const userWithVolunteerAndOrders = await userService.getUserWithVolunteerAndOrders();
+      setUser(userWithVolunteerandOrders);
```
Comment (lerik1408) [inline src/components/profile/Profile.tsx:20]:
```suggestion
      setUser(userWithVolunteerAndOrders);
```

**Pull request merged**